app/src/main/java/org/openimis/imispolicies/RenewList.java

```diff
@@ -530,6 +530,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 finish();
                 return true;
             case R.id.mnuStatistics:
+                ca.CheckInternetAvailable();
```

Comment:
If there is no connectivity, it will not stop the function.

Reply:
Fixed